vendor/github.com/keybase/client/go/libkb/identify_outcome.go

```diff
@@ -225,12 +225,12 @@ func (i IdentifyOutcome) GetErrorAndWarnings(strict bool) (warnings Warnings, er
 
 	if ntf := i.NumTrackFailures(); ntf > 0 {
 		probs = append(probs,
-			fmt.Sprintf("%d track component%s failed",
+			fmt.Sprintf("%d tracked proof%s failed",
```

Comment:
Yes, though I don't think a "followed proof" is a concept that makes sense as much as a "followed person"? I could go either way.